content/about.md

```diff
@@ -13,30 +31,4 @@ Each issue features two articles that speak to a common theme, feature, or conce
 
 *Startwords* (ISSN 2694-2658) is built with the Hugo static site generator using a custom-made theme. In support of open access principles, we publish all articles under a Creative Commons license (CC BY 4.0) and the [website code](https://github.com/Princeton-CDH/startwords) under an Apache 2.0 license.
 
-Though we are not accepting submissions at this time, we make available for reference these working versions of the journal’s [Editorial Guidelines]({{< ref "/editorial-guidelines" >}}) and [Contributor Style Guide]({{< ref "/style-guide" >}}).
-
-{{< wrap class="masthead" >}}
-
-## Masthead
-
-**Editor** Grant Wythoff
-
-**UX Designer** Gissoo Doroudian
-
-**Technical Lead** Rebecca Sutton Koeser
-
-**Manuscript Editing** Camey VanSant
-
-**Technical**
-- Nick Budak (Technical Lead, 2020)
-- Kevin McElwee
-- Grant Wythoff
-
-**Board of Advisors**
-
-- Natalia Ermolaev
-- Zoe LeBlanc
-- Meredith Martin
-- Rebecca Munson (2020)
-
-{{</ wrap >}}
\ No newline at end of file
+Though we are not accepting submissions at this time, we make available for reference these working versions of the journal’s [Editorial Guidelines]({{< ref "/editorial-guidelines" >}}) and [Contributor Style Guide]({{< ref "/style-guide" >}}).
```

Comment:
This removes the `## Masthead` header

Reply:
oops! I meant to put it in the template where I'm rendering contributors, thanks for catching